lib/member_row.rb

```diff
@@ -7,11 +7,15 @@ class MemberRow < Scraped::HTML
   end
 
   field :photo do
-    URI.encode(noko[0].css('img/@src').text)
+    noko[0].css('img/@src').text.gsub(' ', '%20')
```

Comment:
This should be in a separate PR, as it's not directly connected to adding prefixes.